### PR TITLE
chore: added new issue template

### PR DESCRIPTION
**Pull request by cbrissonCoveo**
Added a new template for people to fill when creating a bug report. Let me know if something's missing!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: File a bug report for the ui-kit
```
Comment (olamothe) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]:
```suggestion
description: File a bug report for @coveo/headless, @coveo/atomic or @coveo/quantic
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a bug report, please search for the behaviour in the existing issues. 
+        
+        ---
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: input
```
Comment (olamothe) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: We need an input for "which product is affected". (headless, atomic, quantic)

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    id: version
+    attributes:
+      label: ui-kit version
+      description: Which UI-kit version are you using?
```
Comment (olamothe) [inline .github/ISSUE_TEMPLATE/bug_report.yml:18]: And then the version would need to match one of the above product. `ui-kit` does not have a version, it's only the repo.

**Review by olamothe — COMMENTED** [review]
Thanks !

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: dropdown
+    id: product
+    attributes:
+      label: Which product are you using?
+      options:
+        - Headless
+        - Atomic
+        - Quantic
```
Comment (liviarett) [inline .github/ISSUE_TEMPLATE/bug_report.yml:21]: should we add atomic-react and atomic-angular too?

Comment (liviarett) [reply .github/ISSUE_TEMPLATE/bug_report.yml:21]: maybe not, after seeing Olivier's suggestion for the main description.

**Review by liviarett — APPROVED** [review]

**Review by olamothe — APPROVED** [review]

**Pull request merged**